packages/dev/addons/src/htmlMesh/fitStrategy.ts

```diff
@@ -20,7 +20,7 @@ const FitStrategyContain: FitStrategyType = {
         sizingElement.style.width = `${width}px`;
         sizingElement.style.height = `${height}px`;
 
-        const [childWidth, childHeight] = [scalingElement!.offsetWidth, scalingElement!.offsetHeight];
+        const [childWidth, childHeight] = [scalingElement.offsetWidth, scalingElement.offsetHeight];

```

Comment:
Are unnecessary non-null assertions now flagged?

Reply:
No, i removed the flag. But when I auto-fixed it was on and since it is non-breaking i decided not to revert it. This is one of the flags i would recommend turning on.